### PR TITLE
Add Bookmark endpoints specifications and update TSP version to 0.3.0

### PR DESCRIPTION
**Pull request by kavehshahedi**
The specifications of newly introduced endpoints for managing bookmarks of experiments are added to the API.yaml and API-proposed.yaml. Also, TSP version is udpated to 0.3.0 from 0.2.0.

### CODE REVIEW

File: API.yaml [hunk 1]
```diff
@@ -26,12 +26,14 @@ info:
...
     name: Apache 2
     url: http://www.apache.org/licenses/
-  version: 0.2.0
+  version: 0.3.0
 servers:
 - url: https://localhost:8080/tsp/api
 tags:
 - name: Annotations
   description: Retrieve annotations for different outputs.
+- name: Bookmarks
```
Comment (bhufmann) [inline API.yaml:35]: I think you added it manually and it's generated by the server definitions. See comment here: https://github.com/eclipse-tracecompass-incubator/org.eclipse.tracecompass.incubator/pull/102#pullrequestreview-245536650

Please update the server PR for that.

File: API-proposed.yaml [hunk 2]
```diff
@@ -57,14 +57,193 @@ tags:
 - name: XY
   description: Query XY chart models.
 - name: Bookmarks
-  description: How to bookmark areas of interest in the trace.
+  description: Bookmark areas of interest in the experiment.
```
Comment (bhufmann) [inline API-proposed.yaml:60]: Move Bookmark up after line 34 to match the API.yaml file.

**Review by bhufmann — CHANGES_REQUESTED** [review]

**Review by bhufmann — APPROVED** [review]
Looks good to me. Thanks for the contribution!

**Review by MatthewKhouzam — APPROVED** [review]
In a later version, I would like a notes field that can populate tooltips.

**Pull request merged**